t/132-worker-count.t

```diff
@@ -0,0 +1,34 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+use lib 'lib';
+use Test::Nginx::Socket::Lua;
+
+#worker_connections(1014);
+#master_on();
+#workers(2);
+#log_level('warn');
+
+repeat_each(2);
+#repeat_each(1);
+
+plan tests => repeat_each() * (blocks() * 3);
+
+#no_diff();
+#no_long_string();
+run_tests();
+
+__DATA__
+
+=== TEST 1: content_by_lua
+--- config
+    location /lua {
+        content_by_lua '
```

Comment:
I suggest using `content_by_lua_block` for all new test cases since we already have this handy new feature in master ;)